native/effektio/src/api/message.rs

```diff
@@ -1,100 +1,192 @@
+use log::info;
 use matrix_sdk::{
     deserialized_responses::SyncTimelineEvent,
-    room::Room,
+    room::{
+        timeline::{EventTimelineItem, TimelineItem, TimelineItemContent},
+        Room,
+    },
     ruma::events::{
         room::message::{MessageFormat, MessageType, RoomMessageEventContent},
         AnySyncMessageLikeEvent, AnySyncTimelineEvent, OriginalSyncMessageLikeEvent,
         SyncMessageLikeEvent,
     },
 };
+use std::sync::Arc;
 
 #[derive(Clone, Debug)]
 pub struct RoomMessage {
-    inner: OriginalSyncMessageLikeEvent<RoomMessageEventContent>,
-    room: Room,
-    fallback: String,
+    event_id: String,
```

Comment:
we might want to consider reusing some of the FFI-types in `matrix-sdk-ffi`... they seem very similar and the timeline API of the FFI-crate, too is pretty close to ours...